src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -375,3 +375,6 @@ definitions:
       globalServiceAccountOverride:
         description: Name of an (external) serviceAccount that will be assigned to a dagster user-deployment in favor of creating a new one
         type: string
+      userCodeHelmChartVersion:
```

Comment:
You don't want to make this a required value?

Reply:
As far as I can tell, none of the other values of Dagster are required, so I am okay with this also not being a required value + we have a default for now.

Reply:
Right sorry, this is shared config between the monorepo and scripting, so it can't be mandatory